modules/claims_api/app/controllers/claims_api/v1/forms/intent_to_file_controller.rb

```diff
@@ -63,10 +63,10 @@ def validate
         private
 
         def participant_claimant_id
-          return target_veteran.participant_id unless form_type == 'burial'
-          return target_veteran.participant_id unless header_request?
```

Comment:
this part is important though (at least i think)
burial itf can't, currently, be submitted by either a representative or the Veteran themself
cant be the Veteran for obvious reasons, but can't be the representative because our current implementation of itf uses the Veteran's participant id for the submitter as well.

Reply:
After a lot of back-and-forth between us and Mark, I think we decided to just forbid the action right now if we were not given a `participant_claimant_id`.  [This work](https://github.com/department-of-veterans-affairs/vets-api/pull/7343/commits/a421fbe04d08af8f69e872c2e2b2b19b1d1900ea) provides that logic.